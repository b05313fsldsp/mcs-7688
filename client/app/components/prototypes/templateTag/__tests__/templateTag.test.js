import React from 'react';
import { shallow } from 'enzyme';
import R from 'ramda';
import TemplateTag from '../templateTag';

it('render', () => {
  const wrapper = shallow(<TemplateTag getMessages={R.identity} />);

  expect(wrapper).toMatchSnapshot();
});
