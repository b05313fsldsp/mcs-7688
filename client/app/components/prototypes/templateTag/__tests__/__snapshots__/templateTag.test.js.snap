// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<div
  className="isTemplateTag"
>
  prototypeTemplate
</div>
`;
